client/app/account/billing/autoRenew/update/billing-autoRenew-update.html

```diff
@@ -15,8 +15,8 @@
                                 <tr>
                                     <th class="oui-datagrid__header" tabindex="0" data-translate="autorenew_service_name"></th>
                                     <th class="oui-datagrid__header" tabindex="0" data-translate="autorenew_service_type"></th>
-                                    <th class="oui-datagrid__header text-nowrap" tabindex="0" data-translate="autorenew_service_renew"></th>
                                     <th class="oui-datagrid__header text-nowrap" tabindex="0" data-translate="autorenew_service_frequency"></th>
+                                    <th class="oui-datagrid__header text-nowrap" tabindex="0" data-translate="autorenew_service_renew"></th>
```

Comment:
A quick a11y enhancement would be to add some scope="col":)

In that way you are able to update your first column on your <tbody>:

```html
<th scope="row"
    class="oui-datagrid__cell"
    data-ng-bind="::service.serviceId | sliceContent:25">
</th>```